lib/charms/observability_libs/v0/kubernetes_compute_resources_patch.py

```diff
@@ -399,13 +401,28 @@ def is_ready(self, pod_name, resource_reqs: ResourceRequirements):
             resource_reqs, self.get_actual(pod_name)
         )
 
+    def _handle_pod_termination(self, *args) -> None:
+        logger.debug(
+            "KubernetesComputeResourcesPatch was interrupted by a SIGTERM, likely due to "
+            "pod termination during execution of `config-changed`. Exiting gracefully. "
+            "The hook will be re-run"
```

Comment:
Did you forget to finish the sentence or is it a metaphor for the fact that the charm was killed in flight?
Perhaps it would be helpful to clarify when we _think_ this re-execution will take place?
"The hook that was being ran right now will be re-run by juju as soon as the pod is respawned."